inference-engine/src/gna_plugin/gna_device.hpp

```diff
@@ -49,7 +49,8 @@ class GNADeviceHelper {
     uint64_t instrumentationResults[TotalGna2InstrumentationPoints] = {};
     uint64_t instrumentationTotal[TotalGna2InstrumentationPoints] = {};
     uint32_t instrumentationConfigId = 0;
-
+    uint32_t lastSheduledRequestId;
```

Comment:
Static analysis with complain on uninitialized variable. Need {} or =0 (I would prefer {} to show that the value does not matter)